proto/allocation/allocation.proto

```diff
@@ -34,12 +34,16 @@ message AllocationRequest {
   // If specified, multi-cluster policies are applied. Otherwise, allocation will happen locally.
   MultiClusterSetting multiClusterSetting = 2;
 
+  // Deprecated: Please use gameServerSelectors instead. This field is ignored if the
```

Comment:
As a non-blocking meta note for next time -- I think the changes to the allocation endpoint could probably have been split into a seperate PR from the changes the `GameServerAllocation` resource - just to make the PR a bit smaller and easier to review.